keras_cv/benchmarks/classification_training_benchmark_test.py

```diff
@@ -33,7 +33,7 @@ class ClassificationTrainingBenchmark(
 
     _benchmark_parameters = [
         # TODO(jbischof): revert to ResNetV2 once classification head ready
-        ("ResNet50", models.ResNet50),
+        ("ResNet50", models.ResNet50Backbone),
```

Comment:
Let's just remove this one for now -- this benchmark don't really make sense using a `Backbone`, but it will make sense with an `ImageClassifier` once we have that.

Reply:
Done.